Sources/TuistKit/Mappers/Graph/StaticXCFrameworkModuleMapGraphMapper.swift

```diff
@@ -0,0 +1,182 @@
+import Foundation
+import Path
+import TuistCore
+import TuistSupport
+import XcodeGraph
+
+public final class StaticXCFrameworkModuleMapGraphMapper: GraphMapping {
```

Comment:
I'd add a documentation here explaining what this mapper solves. It might not be obvious from the name.